.github/workflows/continious-integration.yml

```diff
@@ -0,0 +1,164 @@
+name: Continuous Integration
+
+on: pull_request
+
+jobs:
+  phpcs:
+    name: PHP Coding Standards
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          coverage: none
+          tools: composer, cs2pr
+
+      - name: Get Composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Setup cache
+        uses: pat-s/always-upload-cache@v1.1.4
```

Comment:
Why is this in use? This seems work https://github.com/stuttter/ludicrousdb/pull/106/files#diff-a6269b00a8079a2d37d0de82f98d556cR27-R36

Reply:
Check out https://github.com/pat-s/always-upload-cache for the difference

Reply:
No, I did lookup the action, the question is, why is it needed?

Reply:
I haven't run into any particular issues so far, if that's what you mean. But the improvements in that forked version seemed worthwhile. Of course it can be replaced once `actions/cache` is updated.